server/yarn.lock

```diff
@@ -4,7 +4,7 @@
 
 "@apollo/client@3.3.7":
   version "3.3.7"
-  resolved "https://registry.yarnpkg.com/@apollo/client/-/client-3.3.7.tgz#f15bf961dc0c2bee37a47bf86b8881fdc6183810"
+  resolved "https://registry.npmjs.org/@apollo/client/-/client-3.3.7.tgz"
```

Comment:
this is weird. Its either a mistake that is a holdover from using npm in the server directory, or nicolas has changed something about how empirica's yarn is looking for packages, and it's now using the npm registry instead of the yarn registry. 

Can you rebuild your container from scratch and see if this is still there? That'll run `empirica yarn install...` as part of the dockerfile build, and it may revert these changes.